base/deprecated.jl

```diff
@@ -488,6 +488,16 @@ end
     end
 )
 
+if sizeof(Cwchar_t) == 2
+    @deprecate_binding WString UTF16String
+    @deprecate_binding wstring utf16
+	utf16(s::Cwstring) = utf16(convert(Ptr{Cwchar_t}, s))
```

Comment:
I'm going to leave this since the next step is to delete UTF16 and UTF32 anyway.